src/index.js

```diff
@@ -5,7 +5,7 @@ import {cleanup} from './pure'
 // this ensures that tests run in isolation from each other
 // if you don't like this then either import the `pure` module
 // or set the RTL_SKIP_AUTO_CLEANUP env variable to 'true'.
-if (!process.env.RTL_SKIP_AUTO_CLEANUP) {
+if (typeof process === "undefined" || !process.env?.RTL_SKIP_AUTO_CLEANUP) {
```

Comment:
Does your build setup support optional chaining? I hope that's fine 😁

Reply:
Should be transpiled loosely.